src/public/lib/nbgv-python/CHANGELOG.md

```diff
@@ -11,13 +11,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - **BREAKING**: The `version` template field now contains the raw NBGV version string instead of the PEP 440 normalized version. Use `normalized_version` for the PEP 440 compliant string.
 - **BREAKING**: Map unrecognized prerelease labels to `.dev` suffixes to preserve version ordering (e.g., `1.0.0-ci` becomes `1.0.0.dev0+ci`, `1.0.0-ci.1` becomes `1.0.0.dev1+ci`). Complex prerelease tags that cannot be mapped will now raise an error.
+- **BREAKING**: Render boolean version-tuple components as Python boolean literals (`True`/`False`) instead of quoted strings.
 
 ### Fixed
 
+- Avoid accidental path duplication when invoking `nbgv get-version` with a relative project directory
 - Fix `GitVersion` mapping protocol behavior (missing keys now raise `KeyError`; iteration/length reflect all supported keys)
 - Ensure single-element version tuples are rendered with a trailing comma
 - Fix documentation links in project metadata
 - Use strict SemVer 2.0 regex for version parsing
+- Raise `NbgvVersionNormalizationError` when version normalization fails, instead of a generic `RuntimeError`.
```

Comment:
这里写在 Fixed："Raise NbgvVersionNormalizationError ... instead of RuntimeError"。\n但 PR 描述里把它当作 BREAKING CHANGE；对捕获 RuntimeError 的调用方确实是破坏性变更。\n建议在 CHANGELOG 里也标注为 **BREAKING**（放到 Changed 段落），保持语义一致。

Reply:
The project is intended to provide as a hatch plugin instead of a library. From such perspective, it's not a breaking change.